portfolio/src/main/webapp/style.css

```
@@ -1,9 +1,282 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-}
-
-#greeting-container {
-  margin-top: 20px;
-}
+    margin-left: auto;
+    margin-right: auto;
+    width: 1000px;
+  }
+  #whiteCenter {
+    /* display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 50%; */
+    /* text-align: center; */
+    /* display: inline-block; */
+    /* position: relative; */
+    position: absolute;
+    left: 100px;
+    top: 25px;
+    z-index: -1;
+  }
+  #image1 {
+    /* position: relative;
+    top: -38px;
+    left: -400px; */
+    position: absolute;
+    left: -400px;
+    top: -38px;
+    z-index: -1;
+  }
+  #greeting-container {
+    margin-top: 20px;
+    color: rgb(0, 0, 0);
+  }
+  .linkedIn {
+    /*     background:none; */
+    color: yellow;
+    position: relative;
+    left: 240px;
+    top: -10px;
+    width: 100px;
+    font-size: 20px;
+  }
+  .gitHub {
+    border-radius: none;
+    /*     background:none; */
+    color: yellow;
+    position: relative;
+    left: 249px;
+    top: -10px;
+    width: 100px;
+    font-size: 20px;
+  }
+  .Contact {
+    color: yellow;
+    position: relative;
+    left: 260px;
+    top: -10px;
+    width: 100px;
+    font-size: 20px;
+    text-decoration: none;
+  }
+  .ProjectButton {
```

[4.1.4](https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters): Separate words in class names by a hyphen.

```suggestion
  .project-button {
```

Here and elsewhere.